pkgs/top-level/all-packages.nix

```diff
@@ -15941,6 +15941,10 @@ with pkgs;
     patches = config.dwm.patches or [];
   };
 
+  dwm-git = callPackage ../applications/window-managers/dwm/git.nix {
+    patches = config.dwm.patches or [];
```

Comment:
Can you also add?

```nix
conf = config.dwm.conf or [];
```

There might a program called `conf` in future.

Reply:
Good point, done.